app/_hub/kong-inc/ai-semantic-cache/overview/_index.md

```diff
@@ -0,0 +1,133 @@
+    {{site.base_gateway}}/AI Semantic Cache plugin->>Vector database: Store response message options
+    {{site.base_gateway}}/AI Semantic Cache plugin-->>User: Return realtime response
+{% endmermaid %}
+
+### Vector databases
+
+A vector database can be used to store vector embeddings, or numerical representations, of data items. For example, a response would be converted to a numerical representation and stored in the vector database so that it can compare new requests against the stored vectors to find relevant cached items.
+
+Currently, the AI Semantic Cache plugin supports Redis as a vector database.
+
+### Cache management
+
+With the AI Semantic Cache plugin, you can configure a cache of your choice to store the responses from the LLM.
+
+Currently, the AI Semantic Cache plugin supports Redis as a cache.
+
+#### Caching mechanisms
+
+The AI Semantic Cache plugin improves how AI systems provide responses by using two kinds of caching mechanisms:
+
+* **Exact Caching:** This stores precise, unaltered responses for specific queries. If a user asks the same question multiple times, the system can quickly retrieve the pre-stored response rather than generating it again each time. This speeds up response times and reduces computational load.
+* **Semantic Caching:** This approach is more flexible and involves storing responses based on the meaning or intent behind the queries. Instead of relying on exact matches, the system can understand and reuse information that is conceptually similar. For instance, if a user asks about "Italian restaurants in New York City" and later about "New York City Italian cuisine," semantic caching can help provide relevant information based on their related meanings.
+
+Together, these caching methods enhance the efficiency and relevance of AI responses, making interactions faster and more contextually accurate.
+
+### Configuration parameters
+
+You can configure the following required parameters for the AI Semantic Cache plugin:
+
+* `vectordb`: Defines the vector database strategy and settings.
+* `embeddings`: Defines the embeddings provider, model, and configuration
+* `cache_control`: Enables or disables the `cache-control` header handling
+* `storage_ttl`: Sets the `time-to-live` for cached responses
+* `exact_caching`: Enables or disables exact caching
+* `message_countback`: Specifies how many of the user's recent messages will be vectorized for querying the vector database. The entire response is still cached, this setting only determines how many past messages are turned into embeddings.
+* `ignore_system_prompts`: Ignores system messages (`role = system`) in the chat history when sending the embeddings (to generate vectors) if set to `true`. 
+* `ignore_assistant_prompts`: Ignores assistant messages (`role = assistant`) in the chat history when sending the embeddings (to generate vectors) if set to `true`. 
+* `stop_on_failure`: Stops processing if an error occurs, otherwise continues
```

Comment:
You don't need this here at all. This is what's defined in the configuration reference page (/hub/kong-inc/ai-semantic-cache/configuration/), with a lot more detail + nested parameters there.